### PR TITLE
CLOUDP-110845: cannot parse error response

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

_Jira ticket:_ CLOUDP-110845

## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](e2e/E2E-TESTS.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: internal/convert/process_config.go [hunk 1]
```diff
@@ -29,52 +29,53 @@ const (
...
-	TLS                         *TLS                    `yaml:"tls,omitempty" json:"tls,omitempty"`
-	Version                     string                  `yaml:"version,omitempty" json:"version,omitempty"`
-	WiredTiger                  *map[string]interface{} `yaml:"wiredTiger,omitempty" json:"wiredTiger,omitempty"`
+	ArbiterOnly                      *bool                   `yaml:"arbiterOnly,omitempty" json:"arbiterOnly,omitempty"`
+	AuditLogPath                     string                  `yaml:"auditLogPath,omitempty" json:"auditLogPath,omitempty"`
+	AuditLogDestination              string                  `yaml:"auditLogDestination,omitempty" json:"auditLogDestination,omitempty"`
+	AuditLogFormat                   string                  `yaml:"auditLogFormat,omitempty" json:"auditLogFormat,omitempty"`
+	AuditLogFilter                   string                  `yaml:"auditLogFilter,omitempty" json:"auditLogFilter,omitempty"`
+	BackupRestoreCheckpointTimestamp interface{}             `yaml:"backupRestoreCheckpointTimestamp,omitempty" json:"backupRestoreCheckpointTimestamp,omitempty"`
```
Comment (gssbzn) [inline internal/convert/process_config.go:37]: adding this field but gofmt reformat the whole struct

File: internal/convert/process_config.go [hunk 2]
```diff
@@ -151,31 +152,32 @@ func newLogRotate() *opsmngr.LogRotate {
...
+		BindIP:                           p.Args26.NET.BindIP,
+		BindIPAll:                        p.Args26.NET.BindIPAll,
+		IPV6:                             p.Args26.NET.IPV6,
+		ProcessType:                      p.ProcessType,
+		Version:                          p.Version,
+		FeatureCompatibilityVersion:      p.FeatureCompatibilityVersion,
+		Hostname:                         p.Hostname,
+		Name:                             p.Name,
+		SetParameter:                     p.Args26.SetParameter,
+		BackupRestoreCheckpointTimestamp: p.BackupRestoreCheckpointTimestamp,
```
Comment (gssbzn) [inline internal/convert/process_config.go:180]: adding this, the rest is gofmt

File: internal/store/agents.go [hunk 3]
```diff
@@ -54,7 +55,7 @@ type AgentAPIKeyDeleter interface {
 func (s *Store) Agents(projectID, agentType string) (*opsmngr.Agents, error) {
 	switch s.service {
 	case config.OpsManagerService, config.CloudManagerService:
-		result, _, err := s.client.(*opsmngr.Client).Agents.ListAgentsByType(s.ctx, projectID, agentType)
+		result, _, err := s.client.(*opsmngr.Client).Agents.ListAgentsByType(s.ctx, projectID, agentType, nil)
```
Comment (gssbzn) [inline internal/store/agents.go:58]: there's a separate ticket to add pagination here

**Review by colm-quinn — APPROVED** [review]
LGTM, thanks for the quick fix!

**Pull request merged**